downstream/modules/automation-hub/proc-create-synclist.adoc

```diff
@@ -28,5 +28,5 @@ All {CertifiedName} are included by default in your initial organization synclis
 . Log in to `{Console}`.
 . Navigate to menu:Automation Hub[Collections].
 . Use the toggle switch on each collection to determine whether to exclude it from your synclist.
-. When you finish managing collections for your synclist, navigate to menu:Automation Hub[Repo Management] to initiate the remote repository synchronization to your {PrivateHubName}. 
-. Optional: If your remote repository is already configured, you can manually synchronize Red Hat {CertifiedName} to your {PrivateHubName} to update the collections content that you made available to local users.
\ No newline at end of file
+. When you finish managing collections for your synclist, navigate to menu:Automation Hub[Repositories] to initiate the remote repository synchronization to your {PrivateHubName}.
```

Comment:
We're going to be deprecating synclists in automation hub. If a user wants to select a subset of the collections in console.redhat.com, they should add a requirements.yaml to their rh-certified remote. It might be worth it to remove this from the documentation now.

https://docs.google.com/presentation/d/1njlPBceZTCFYx2Fr2ehtEfOfuRggpBK66mnSqkGZzLI/edit#slide=id.g24f207286f5_19_11